lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -285,11 +392,119 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
         this.categoryControlVisible = false;
     }
 
-    cancelChanges() {
+    cancelChanges(buttonType: ButtonType, event: MouseEvent, group?: CardViewGroup) {
+        event.stopPropagation();
+        this.toggleEditMode(buttonType, group);
         this.revertChanges();
         this.loadProperties(this.node);
     }
 
+    // Returns the editing state of the panel
+    isEditingPanel(): boolean {
+        return (
+            (this.editable && this.hasMetadataChanged) ||
```

Comment:
This can be way shorter by going through editable conditions inside parentheses with `||` and adding final `&& this. hasMetadataChanged` at the end

Reply:
Fixed.